lib/conjur/cli.rb

```diff
@@ -69,6 +71,17 @@ def run args
         super args
       end
 
+      def install_core_plugins
+        to_install = CORE_PLUGINS - Conjur::Config.plugins
+        unless to_install.empty? or ENV['CONJUR_SKIP_PLUGINS']
+          puts 'Installing core plugins'
+          to_install.each do |plugin|
+            puts "- #{plugin}"
+            self.run ['plugin', 'install', plugin]
```

Comment:
This is going to fail in customer environments where access to Rubygems is prohibited. Such as Box.